easybuild/easyblocks/generic/conda.py

```diff
@@ -44,6 +46,7 @@ def extra_options(extra_vars=None):
         """Extra easyconfig parameters specific to Conda easyblock."""
         extra_vars = Binary.extra_options(extra_vars)
         extra_vars.update({
+            'conda_cmd': [DEFAULT_CONDA_CMD, "Conda command to use", CUSTOM],
```

Comment:
Are `conda` and `mamba` on par w.r.t. supported options?

I'm just wondering if a `use_mamba` easyconfig parameter that can be set to `True` would be a better option here, not sure.

Reply:
The `mamba` docs say yes: https://mamba.readthedocs.io/en/latest/user_guide/mamba.html#mamba-vs-conda-clis

Reply:
`micromamba` seems to support everything currently in the easyblock: https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html#quickstarts

Reply:
If you were being picky you could add a list of supported Conda commands, but I think that is overkill right now

Reply:
Rather than set conda vs mamba via a parameter, should be not detect which is available as a dependency and use it?